packages/plugin-ui/src/PluginUI.tsx

```diff
@@ -90,6 +95,18 @@ export const PluginUI = (props: PluginUIProps) => {
             Copy
           </button>
         </div> */}
+          {warnings.length > 0 && (
+            <div className="flex flex-col bg-red-700 text-white p-4 w-full">
+              <h3 className="text-lg">Warnings:</h3>
+              <ul className="list-disc list-inside">
```

Comment:
needs to support both light and dark mode, I would do something like bg-red-300 text-black dark:bg-red-700 dark:text-white, or use opacity like dark:bg-red-700/40

Reply:
I would probably also use a lucide warning icon and maybe yellow color since it is not THAT bad, just a warning, not an error